yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -221,6 +224,15 @@ export const deployL1Contracts = async (
 
   logger.info(`Deployed Gas Portal at ${gasPortalAddress}`);
 
+  // fund the rollup contract with gas tokens
+  const gasToken = getContract({
+    address: gasTokenAddress.toString(),
+    abi: contractsToDeploy.gasToken.contractAbi,
+    client: walletClient,
+  });
+  await gasToken.write.mint([rollupAddress.toString(), 100000000000000000000n], {} as any);
```

Comment:
Remember to `waitForTransactionMined` (or however is the helper called in viem), otherwise you can run into race conditions